### ISSUE TITLE
 false alarm

### ISSUE THREAD

**Issue by komatera** [MetaMask/eth-phishing-detect#3690]
![photo_2020-02-12_10-31-03](https://user-images.githubusercontent.com/50356972/74312455-c9814180-4d82-11ea-9ad8-2984afc980fc.jpg)
![photo_2020-02-12_10-30-59](https://user-images.githubusercontent.com/50356972/74312458-ca19d800-4d82-11ea-8862-cb3084c6486e.jpg)
![photo_2020-02-12_10-30-58](https://user-images.githubusercontent.com/50356972/74312459-cab26e80-4d82-11ea-9641-cd036fa7ac35.jpg)
Hello! I am the worker of TASS (Info Agency), and our site is blocked by metamask, but it isn't etherium site and it's not similar to oasis.app, like it says. Pls, unblacklist tass.com and tass.ru.

Another issue is that on my PC tass.ru is ok, om my phone tass.ru is blacklisted. How is it possible?

These sites are just news sites.

**Comment by komatera** [comment]
Wow, tass.ru isn't blocked, but Tass.ru (big letter first) is blocked. wtf?

**Comment by xrn** [comment]
Already solved
